src/content/api/module-methods.md

```diff
@@ -73,11 +73,21 @@ W> This feature relies on [`Promise`](https://developer.mozilla.org/en-US/docs/W
 The spec for `import` doesn't allow control over the chunk's name or other properties as "chunks" are only a concept within webpack. Luckily webpack allows some special parameters via comments so as to not break the spec:
 
 ``` js
+// single target
 import(
   /* webpackChunkName: "my-chunk-name" */
   /* webpackMode: "lazy" */
   'module'
 );
+
+// multiple possible targets
+import(
+  /* webpackInclude: "\\.json$" */
```

Comment:
the `webpackInclude` and `webpackExclude` values use javascript regex syntax.

EG:
``` js
/* webpackInclude: /\.json$/ */
```

Reply:
the original PR used string literals but later changed to support this syntax.

Reply:
great catch, changing...